tools/bots/tsconfig.json

```diff
@@ -0,0 +1,11 @@
+{
+	"compilerOptions": {
+		"baseUrl": ".",
+		"paths": {
+			"web3": ["node_modules/web3/dist/web3"],
```

Comment:
Why do you specify the paths? it takes them automatically in the node_modules by default

Reply:
It is supposed to do so, but when compiling Typescript I got these errors:

```shell
mission-control-bot.ts(1,47): error TS2792: Cannot find module 'discord.js'. Did you mean to set the 'moduleResolution' option to 'node', or to add aliases to the 'paths' option?
mission-control-bot.ts(2,18): error TS2792: Cannot find module 'web3'. Did you mean to set the 'moduleResolution' option to 'node', or to add aliases to the 'paths' option?
```
So I ended up defining those aliases with the `paths` option.